build.sbt

```diff
@@ -205,6 +205,16 @@ lazy val mimaSettings = Seq(
     }.toSet
   },
   mimaBinaryIssueFilters ++= Seq(
+    ProblemFilters.exclude[DirectMissingMethodProblem]("fs2.Stream.PurePipeOps"),
```

Comment:
To preserve binary compatibility, we should deprecate this stuff until 3.0 release -- I think that will coincide with cats-effect 3, probably in the 6+ month timeframe.